src/common/constants.ts

```diff
@@ -1,4 +1,5 @@
 const MAP_API_KEY = process.env.GATSBY_MAP_API_KEY
+const GROWTHBOOK_CLIENT_KEY = process.env.GROWTHBOOK_CLIENT_KEY || 'sdk-0aHHaxKhwn0bP1HE'
```

Comment:
That's a fake fallback API Key, Right? 🤔

Reply:
It's our current demo growthbook account. Do we have a way of setting default environment variables for when a new developer checks out the repo?

Reply:
I don't know if we have development env file in deriv-com 🤔 Maybe @habib-deriv can answer that 🙇🏻